src/main/scala/ai/privado/languageEngine/java/threatEngine/CustomPrivacyLoggerMustbeUsed.scala

```diff
@@ -37,7 +37,7 @@ object CustomPrivacyLoggerMustbeUsed {
               violatingFlows
                 .append(
                   ViolationProcessingModel(
-                    leakage.name,
+                    s"${leakage.methodFullName}::${leakage.file.head.name}",
```

Comment:
Always better to use things like `leakage.file.name.headOption.getOrElse(Constants.Unknown)`